### PR TITLE
Fixed a bug that stopped teams switching at halftime

### PR DESCRIPTION
**Pull request by clarkeyio**
Set up a halftime event that triggers the teams to switch at halftime, if manual teams are being used.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 *.zip
 *.smx
 builds
+.atom-build.*
```
Comment (splewis) [inline .gitignore]: Can you revert the change to the .gitignore file? I'd prefer to keep any editor specific things out.

Comment (clarkeyio) [inline .gitignore]: Sure!

**Comment by splewis** [comment]
Tested this now - worked in my test. Thanks.
